template/tasks.py

```diff
@@ -138,8 +138,9 @@ def test(ctx, name="", verbose=False, suite=None):
     """
     Runs the test suite
     """
-
-    ctx.run("poetry run playwright install")  # ensure we have a playwright browser
+    if suite != "system":
+        # Ensure we have a playwright browser; Skip this when running system suite only.
+        ctx.run("poetry run playwright install")
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** We need to account for when suite is not passed at all

The `suite` parameter can be `None` so I'd suggest that when it _is_ that we still try to install playwright. Something like:

```python
if suite in (None, "functional"):
    ctx.run("poetry run playwright install")
```
‏

Reply:
I removed this chunk from the `test` task in favour of the new `install` task.